docs/getting_started.md

```diff
@@ -36,7 +36,7 @@ cd /var/lib/libvirt/images && \
 virt-install \
     --name ${VMNAME} \
     --vcpus 2 \
-    --memory 2048 \
+    --memory 3072 \
```

Comment:
My attempt failed at this step. My hypervisor is Fedora, and I've been able to use it to create VMs for development. Those use a different kickstart file than the one mentioned in the instructions referenced here, so perhaps that's the difference.

When I follow the instructions for creating a VM, it progresses for a while and then reboots and fails with dracut-initqueue timeout errors before it finally enters the emergency shell.

```
It seems that the boot has failed. Possible causes include
missing inst.stage2 or inst.repo boot parameters on the
kernel codlin. Please verify that you have specified
inst.stage2 or inst.repo.
```

Reply:
Does the KICKSTART value need to be a URL, then? I tried pointing it to the local file on my filesystem.

Reply:
Yes, it must be URL because the VM cannot access local hypervisor files after the boot.

Reply:
That's working much better.